mirrord/layer/src/file/filter/read_local_by_default.rs

```diff
@@ -92,5 +92,13 @@ pub fn regex_set_builder() -> RegexSetBuilder {
         patterns.push(format!("{}$", regex::escape(&executable.to_string_lossy())));
     }
 
+    // Hidden files and directories in $HOME.
+    if let Ok(home) = env::var("HOME") {
+        patterns.push(format!(
+            "^{}/\\.",
```

Comment:
```suggestion
            "^{}/\.",
```

Otherwise this requires a `\` in the path, doesn't it?